src/coreclr/jit/emitxarch.cpp

```diff
@@ -128,6 +139,122 @@ bool emitter::IsAVXInstruction(instruction ins) const
     return UseVEXEncoding() && IsSSEOrAVXInstruction(ins);
 }
 
+bool emitter::IsAVX512Instruction(instruction ins) const
+{
```

Comment:
Might be good to short circuit this as:
```cpp
if (!UseEvexEncoding())
{
    return false;
}

...
```

That would remove doing "extra" work in the case where no EVEX support exists.

Reply:
Changed.